test/e2e/framework/framework.go

```diff
@@ -306,7 +306,7 @@ func createNamespace(client kubeclientset.Interface, baseName string) (string, e
 	// TODO(marun) should all api calls be made 'robustly'?
 	var namespaceName string
 	if err := wait.PollImmediate(PollInterval, TestContext.SingleCallTimeout, func() (bool, error) {
-		namespace, err := client.Core().Namespaces().Create(namespaceObj)
+		namespace, err := client.CoreV1().Namespaces().Create(namespaceObj)
```

Comment:
I get that V1 should be preferred, but how does golint know that?

Reply:
The function gets a deprecated comment.

```go
	// Deprecated: please explicitly pick a version if possible.
	Core() corev1.CoreV1Interface
```